### ISSUE TITLE
`unimported` gives error when no `ignore_subdirectories` specified

### ISSUE THREAD

**Issue by extrange** [beetbox/beets#4657]
The `unimported` plugin errors when running unconfigured.

### Problem

`beet config -e`:

```yaml
plugins: unimported
```

When running `beet unimported`, I get:

```bash
configuration error: unimported.ignore_subdirectories not found
```

This can be fixed by adding `ignore_subdirectories: ''` to `beet config -e`, however the plugin should not require this to be specified.

**Comment by sampsyo** [comment]
That's odd! It looks like the plugin is providing a default for that option:
https://github.com/beetbox/beets/blob/6aa562e2aee4dd95a231ba8043ca4ff924ec2ddf/beetsplug/unimported.py#L36

Could you please include some verbose output?

**Comment by extrange** [comment]
This the output of beet -v unimported:

```
user configuration: /root/.config/beets/config.yaml
data directory: /root/.config/beets
plugin paths: 
Sending event: pluginload
library database: /data/musiclibrary.db
library directory: /music
Sending event: library_opened
configuration error: unimported.ignore_subdirectories not found
```

`config.yaml`:

```
plugins:
  - chroma
  - edit
  - fetchart
  - fromfilename
  - replaygain
  - unimported

directory: /music
library: /data/musiclibrary.db

badfiles:
  check_on_import: yes

replaygain:
  backend: ffmpeg
```

I'm not sure why this doesn't happen with `ignore_extensions`.

**Comment by jackwilsdon** [comment]
What version of beets are you on? Seems like this was a bug in 1.6.0 that was fixed in #4179.

You can either [run the latest source version of beets](https://docs.beets.io/en/latest/faq.html#run-the-latest-source-version-of-beets) (be sure to make backups!) or set the option manually:

```YAML
unimported:
  ignore_subdirectories: []
```

**Comment by extrange** [comment]
Oh right. I'm using 1.6.0 - that's probably the reason then.
